db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+aurora = Shelter.create!(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
```

Comment:
It might be helpful for your deployment to add `destroy_all` commands at the top for every object, so that you can run `rails db:seed` on Heroku multiple times and not need to worry about duplicate data

example:
`Pet.destroy_all`
`Application.destroy_all`
...etc

Reply:
Got it! I would love to discuss this briefly with you live just to get a bit more clarification what that actually does/looks like in the application.

Reply:
Absolutely! We can definitely take a look at this in your next project. Your seeds file there should also have destroy  commands first so as not to duplicate data